linux_os/guide/system/crypto_policies/bind_obeys_crypto_policy/oval/shared.xml

```diff
@@ -0,0 +1,28 @@
+<def-group>
+  <definition class="compliance" id="bind_obeys_crypto_policy" version="1">
+    <metadata>
+      <title>The BIND library obeys system-wide crypto policy.</title>
+      <affected family="unix">
+        <platform>multi_platform_fedora</platform>
+      </affected>
+      <description>The BIND library shouldn't override system-wide crypto policy setting.</description>
+    </metadata>
+    <criteria>
+      <criterion test_ref="test_bind_obeys_crypto_policy"
+      comment="Check that the configuration includes the policy config file." />
+    </criteria>
+  </definition>
+
+  <ind:textfilecontent54_test id="test_bind_obeys_crypto_policy"
+  comment="Check that the configuration includes the policy config file."
+  check="all" check_existence="all_exist" version="1">
+    <ind:object object_ref="object_bind_obeys_crypto_policy" />
+  </ind:textfilecontent54_test>
+
+  <ind:textfilecontent54_object id="object_bind_obeys_crypto_policy"
+  version="1">
+    <ind:filepath>/etc/named.conf</ind:filepath>
```

Comment:
Consider anchoring the filepath with `^` and `$`.

Reply:
My mistake, there is no need for anchors here as `filepath` element is targeting a specific file and not doing a regular expression matching to grab existing files.